### PR TITLE
Add trello webhook callback

### PR DESCRIPTION
**Pull request by dmitrytrager**
need some changes on the server to play with real trello callbacks

### CODE REVIEW

File: app/interactors/log_trello_event.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class LogTrelloEvent
+  include Interactor
+
+  def perform
+    ActiveSupport::Notifications.instrument(
+      'trello.event',
+      action: action.attributes,
+      model: model.attributes
+    )
```
Comment (VladimirMikhailov) [inline app/interactors/log_trello_event.rb]: Should we use the same syntax here as [below](https://github.com/fs/tt-ruby/pull/11/files#diff-ad6b8dc8196f0fa0e36eaf362683bc8bR9)?

File: app/models/trello_entity.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  include ActiveModel::Validations
+  include ActiveModel::Conversion
+  extend ActiveModel::Naming
+
+  attr_reader :attributes
+
+  def self.field(name)
+    define_method(name) { @attributes[name] }
+    define_method("#{name}=") { |value| @attributes[name] = value }
+  end
```
Comment (VladimirMikhailov) [inline app/models/trello_entity.rb:11]: AFAICS, looks like an alias for `attr_accessor` but with different implementation.

**Comment by VladimirMikhailov** [comment]
+

File: app/models/trello_entity.rb (hunk 2 above)
Comment (dmitrytrager) [inline app/models/trello_entity.rb:11]: I see, this looks similar, but implemented with `@attributes`

**Comment by VladimirMikhailov** [comment]
:+1:

**Pull request merged**